app/controllers/services_controller.rb

```diff
@@ -95,11 +95,11 @@ def create
           end
         end
       else
-        flash[:error] =  service_route.capitalize + ' returned invalid data for the user id.'
+        flash[:alert] =  service_route.capitalize + ' returned invalid data for the user id.'
         redirect_to new_user_session_path
       end
     else
-      flash[:error] = 'Error while authenticating via ' + service_route.capitalize + '.'
+      flash[:alert] = 'Error while authenticating via ' + service_route.capitalize + '.'
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.